src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -16,92 +16,92 @@
-  }
-
-  @DontRun(env = {"sandbox"})
-  @Test(groups = {"ChatTestsStaff_002", "ChatStaff"})
-  public void ChatTestsStaff_002_verifySwitchingBetweenMainAndPrivateSections() {
-    switchToWindow(driverOne);
-    ChatPageObject chatUserOne =
-        openChatForUser(driverOne, credentials.userNameStaff, credentials.passwordStaff);
-
-    switchToWindow(driverTwo);
-    ChatPageObject chatUserTwo =
-        openChatForUser(driverTwo, credentials.userNameStaff2, credentials.passwordStaff2);
-
-    String userTwoMessage = chatUserTwo.generateMessageFromUser(credentials.userNameStaff2);
-    chatUserTwo.writeOnChat(userTwoMessage);
-
-    switchToWindow(driverOne);
-    chatUserOne.verifyMessageOnChat(userTwoMessage);
-
-    chatUserOne.selectPrivateMessageToUser(credentials.userNameStaff2);
-    chatUserOne.verifyPrivateMessageHeader();
-    chatUserOne.verifyPrivateMessageIsHighlighted(credentials.userNameStaff2);
-    chatUserOne.verifyPrivateChatTitle();
-
-    chatUserOne.clickOnMainChat();
-    chatUserOne.verifyMainChatIsHighlighted();
-    chatUserOne.verifyMessageOnChat(userTwoMessage);
-  }
-
-  @DontRun(env = {"sandbox"})
-  @Test(groups = {"ChatTestsStaff_003", "ChatStaff"})
-  public void ChatTestsStaff_003_sendPrivateMessage() {
-    switchToWindow(driverOne);
-    ChatPageObject chatUserOne =
-        openChatForUser(driverOne, credentials.userNameStaff, credentials.passwordStaff);
-
-    switchToWindow(driverTwo);
-    ChatPageObject chatUserTwo =
-        openChatForUser(driverTwo, credentials.userNameStaff2, credentials.passwordStaff2);
-
-    String userTwoPublicMessage = chatUserTwo.generateMessageFromUser(credentials.userNameStaff2);
-    chatUserTwo.writeOnChat(userTwoPublicMessage);
-
-    switchToWindow(driverOne);
-    chatUserOne.verifyMessageOnChat(userTwoPublicMessage);
-
-    switchToWindow(driverTwo);
-    String userTwoPrivateMessage = chatUserTwo.generateMessageFromUser(credentials.userNameStaff2);
-    chatUserTwo.selectPrivateMessageToUser(credentials.userNameStaff);
-    chatUserTwo.writeOnChat(userTwoPrivateMessage);
-
-    switchToWindow(driverOne);
-    chatUserOne.verifyPrivateMessageHeader();
-    chatUserOne.verifyPrivateMessageNotification();
-    chatUserOne.clickOnUserInPrivateMessageSection(credentials.userNameStaff2);
-    chatUserOne.verifyMessageOnChat(userTwoPrivateMessage);
-  }
-
-  @DontRun(env = {"prod", "sandbox"})
-  @Test(groups = {"ChatTestsStaff_004", "ChatStaff"})
-  public void ChatTestsStaff_004_basicUserChatFails() {
-
-    switchToWindow(driverOne);
-    ChatPageObject chatUserOne =
-        openChatForUser(driverOne, credentials.userName10, credentials.password10);
-    SpecialPageObject special = new SpecialPageObject(driver);
-    special.verifyPageHeader("Permissions error");
-  }
+//  @DontRun(env = {"sandbox"})
+//  @Test(groups = {"ChatTestsStaff_001", "ChatStaff"})
+//  public void ChatTestsStaff_001_twoUserEnterChat() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)